src/api/l_graphics_buffer.c

```diff
@@ -146,8 +146,8 @@ void luax_readbufferdata(lua_State* L, int index, Buffer* buffer, char* data) {
   uint32_t dstIndex = luax_optu32(L, index + 2, 1) - 1;
 
   if (blob) {
-    uint32_t limit = MIN(blob->size / stride - srcIndex, info->length - dstIndex);
-    uint32_t count = luax_optu32(L, index + 3, limit);
+    graphics_size limit = (graphics_size)(MIN(blob->size / stride - srcIndex, info->length - dstIndex));
+    graphics_size count = luax_optgraphics_size(L, index + 3, limit);
```

Comment:
Same sort of comment here.  I don't think `count` should be a `graphics_size` since it's not a number of bytes.  At the point these quantities are turned into sizes (basically post-multiplication down below), graphics_size would become the correct type.